### PR TITLE
Problem: missing erc 1155 nft download & display #78

### PR DESCRIPTION
**Pull request by leejw51crypto**
add erc 1155 nft download and display

### CODE REVIEW

**Comment by leejw51crypto** [comment]
image

**Review by tomtau — COMMENTED** [review]
why is the old NFT map deleted? could the demo contain both? oh, it's now called NFTErc721.umap? it's hard to know from the GitHub diff

**Review by tomtau — APPROVED** [review]
ok if old nft demo is still there

**Review by damoncro — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[submodule "play-unreal-plugin"]
-	path = play-unreal-plugin
-	url = git@github.com:cronos-labs/play-unreal-plugin.git 
-	branch = main 
+[submodule "Plugins/play-unreal-plugin"]
+	path = Plugins/play-unreal-plugin
+	url = git@github.com:leejw51crypto/play-unreal-plugin.git
```
Comment (tomtau) [inline .gitmodules:3]: do you need some patch in the plugin or why the personal fork?

Comment (leejw51crypto) [reply .gitmodules:3]: i'll change to public repo

**Comment by leejw51crypto** [comment]
changed folder structure , submit review again

**Pull request closed without merging**